nuxt-frontend/components/flutter-app-development/CtaSection.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <section>
+    <div class="relative h-[40vh] lg:h-[65vh] 2xl:h-[670px]">
```

Comment:
compare this cta's height with other page cta its too much bigger cta should be of same height everywhere